lib/webdriver.py

```diff
@@ -150,8 +162,18 @@ def _get_driver(self) -> Chrome:
 
     def _set_headers_from_request(self, driver: Chrome) -> None:
         # Retrieving the headers could fail if the form isn't given enough time to submit
-        time.sleep(10)
+        WebDriverWait(driver, 60).until(
```

Comment:
> Is this necessary to wait for anymore since we are already waiting for the response in the above changes? I don’t see a point in waiting if we already have the valid response

Realized what the issue was. Without the wait, the script would still crash on this line:

`flights = json.loads(driver.requests[1].response.body)["upcomingTripsPage"]`

I erroneously added a wait to the wrong section. I've implemented a fix by waiting for the second request (and response) to retrieve flights before we try to load that response.